### PR TITLE
Add BC breaks information for dashboard modules and other notable changes for 1.7.6.6

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Add information about the security issue: https://github.com/PrestaShop/PrestaShop/security/advisories/GHSA-mc98-xjm3-c4fm
| Type?         | improvement

### CODE REVIEW

File: src/content/1.7/modules/core-updates/1.7.6.md [hunk 1]
```diff
@@ -92,4 +92,6 @@ PhpExcel (no longer supported) has been replaced by PHPSpreadsheet – ([See rel
   - `src/Core/Addon/Module/ModuleManager.php::_construct()` receives an instance of `CacheClearerInterface` instead of `CacheClearer` – ([See related change](https://github.com/PrestaShop/PrestaShop/pull/12022)).
   - Permissions for files created by PrestaShop are no longer set to 0777. Permissions are now set to 0755 for directories and 0644 for files.
   - To prevent having jQuery included twice, It is no longer possible to add jQuery in BO controllers based on the new theme using addJquery() – ([See related change](https://github.com/PrestaShop/PrestaShop/pull/12716)).
+- **Module:**
+  - Dashboard modules can no longer use `AdminDashboardController::ajaxProcessSaveDashConfig()` to save values without implementing the `saveDashConfig` method and if needed the `validateDashConfig` method. This is not possible anymore in PrestaShop `1.7.6.6` in order to enforce the shop's security.
```
Comment (matks) [inline src/content/1.7/modules/core-updates/1.7.6.md:96]:
```suggestion
  - Dashboard modules can no longer use `AdminDashboardController::ajaxProcessSaveDashConfig()` to save values without implementing the `saveDashConfig` method and if needed the `validateDashConfig` method. This is not possible anymore from PrestaShop `1.7.6.6` in order to enforce the shop's security.
```

**Comment by matks** [comment]
Should we explain _all_ notable and risky (BC break) changes from https://build.prestashop.com/news/prestashop-1-7-6-6-maintenance-release/ into this doc ?

**Comment by PierreRambaud** [comment]
> Should we explain _all_ notable and risky (BC break) changes from https://build.prestashop.com/news/prestashop-1-7-6-6-maintenance-release/ into this doc?

Yes, we need to do it.

**Comment by matks** [comment]
Thank you @PierreRambaud

**Pull request merged**